torchani/cuaev/setup.py

```diff
@@ -0,0 +1,49 @@
+import os
+import torch
+import subprocess
+from setuptools import setup
+from torch.utils.cpp_extension import BuildExtension, CUDAExtension
+
+nvcc_args = ["-gencode=arch=compute_50,code=sm_50", "-gencode=arch=compute_60,code=sm_60",
+             "-gencode=arch=compute_61,code=sm_61", "-gencode=arch=compute_70,code=sm_70",
+             "-Xptxas=-v", '--expt-extended-lambda', '-use_fast_math']
+include_dirs = []
+cuda_version = float(torch.version.cuda)
+if cuda_version >= 10:
+    nvcc_args.append("-gencode=arch=compute_75,code=sm_75")
+if cuda_version >= 11:
+    nvcc_args.append("-gencode=arch=compute_80,code=sm_80")
+if cuda_version >= 11.1:
+    nvcc_args.append("-gencode=arch=compute_86,code=sm_86")
+if cuda_version < 11:
+    # if no cub, download it to include dir from github
+    if not os.path.isdir('./include/cub'):
+        if not os.path.exists('./include'):
+            os.makedirs('include')
+        commands = """
+        echo "Downloading CUB library";
+        wget -q https://github.com/NVIDIA/cub/archive/main.zip;
+        unzip -q main.zip -d include;
+        mv include/cub-main/cub include;
+        echo "Removing unnecessary files";
+        rm main.zip;
+        rm -rf include/cub-main;
+        """
+        subprocess.run(commands, shell=True, check=True, universal_newlines=True)
+    include_dirs.append(os.path.abspath("./include"))
+
+setup(
+    name='cuaev',
+    version='0.1',
+    ext_modules=[
+        CUDAExtension(
+            name='cuaev',

```

Comment:
It's very minor I guess but can I suggest to move the `cuaev` module into `torchani.cuaev` so it's scoped within `torchani`? The python code using `cuaev` will need to be fixed too.

Reply:
This `setup.py` will be eventually abandoned. So please ignore this file for now.

Reply:
In the last commit I am just renaming it, so that it will conflict with https://github.com/aiqm/torchani/pull/521